README.md

```diff
@@ -14,10 +14,22 @@ manipulate complex JSON and YAML data models, integrate disparate
 tooling, and provides quick access to aggregate views of operational
 data.
 
-- [Install a prebuilt version](docs/installation.md)
-- [Clone and contribute!](docs/dev/README.md)
+## Install
 
-## The Kui Experience
+The fastest option is to install a prebuilt release.
+
+[Kui-MacOS.tar.bz2](https://macos-tarball.kui-shell.org) **|** [Kui-Linux.zip](https://linux-zip.kui-shell.org)
+
+Visit [this section](docs/installation.md) to know
+more about the setup or if you need alternative
+methods (ie: headless).
```

Comment:
alternative *installation* methods.

(two changes there, add the "installation" and remove the (ie: headless) part